example/jvm/src/main/scala/hedgehog/examples/state/TurnstileTest.scala

```diff
@@ -37,7 +37,7 @@ object TurnstileTest extends Properties {
     List(
       property("sequential", testSequential)
 
-    /**
+    /*
```

Comment:
Changed because of the following error when generating scaladoc
```
[error] scala-hedgehog/example/jvm/src/main/scala/hedgehog/examples/state/TurnstileTest.scala:40:5: discarding unmoored doc comment
[error]     /**
[error]     ^
[info] No documentation generated with unsuccessful compiler run
[error] one error found
```

Reply:
Separate PR?

Reply:
Hmmm this one was the cause of the error when I tested it locally by running `packagedArtifacts`. Let me check it again.

Reply:
This was caused by missing `noPublish` settings in the `example` project.
Now it's fine.
https://github.com/hedgehogqa/scala-hedgehog/blob/92a38025a22b68c82233dedc90b510551b8c5b42/build.sbt#L52